multimodal/tests/unittests/others_2/test_zero_shot.py

```diff
@@ -8,12 +10,12 @@
 
 def download_sample_images():
     url = "http://images.cocodataset.org/val2017/000000039769.jpg"
-    image = Image.open(requests.get(url, stream=True).raw)
+    image = Image.open(BytesIO(requests.get(url).content))
     cat_image_name = "cat.jpg"
     image.save(cat_image_name)
 
-    url = "https://farm1.staticflickr.com/29/57154382_07b25134f7_z.jpg"
-    image = Image.open(requests.get(url, stream=True).raw)
```

Comment:
This line was failing with status 403 returned by `requests.get(url, stream=True)`